### ISSUE TITLE
GraphQL item mutation broken in 9.18.0

### ISSUE THREAD

**Issue by stx-chris** [directus/directus#15691]
### Describe the Bug

When a mutation is executed, the request fails with an error that the (auto-generated) id field must be provided. 

### To Reproduce

1. Create a collection "test" with a an auto-generated id field and the suggested system fields
2. Try to send the following GraphQL mutation:

    ```gql
    mutation {
      create_test_item(data: { status: "Draft" }) {
        id 
        status 
      }
    }
    ```

3. Receive error in response.

Note that 9.17.4 didn't have this error.

### Errors Shown

```js
{
    "errors": [
        {
            "message": "GraphQL validation error.",
            "extensions": {
                "code": "GRAPHQL_VALIDATION_EXCEPTION",
                "graphqlErrors": [
                    {
                        "message": "Field \"create_test_input.id\" of required type \"ID!\" was not provided.",
                        "locations": [
                            {
                                "line": 2,
                                "column": 34
                            }
                        ]
                    }
                ]
            }
        }
    ]
}
```

### What version of Directus are you using?

9.18.0

### What version of Node.js are you using?

16.13.2

### What database are you using?

Postgres 13

### What browser are you using?

Chrome, Safari

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Might be related to #15465.

**Comment by sdekna** [comment]
I am having the exact same issue... cannot create any new item via mutation unless I provide an id, which should be automatically created.

Any fix yet?

**Comment by licitdev** [comment]
@sdekna It has been patched and released in `9.18.1`.
